pom.xml

```diff
@@ -77,7 +77,7 @@
 
     <!-- Versions -->
     <java.level>1.8</java.level>
-    <maven.version>3.2.5</maven.version>
+    <maven.version>3.0.5</maven.version>
```

Comment:
reverting this back to 3.2.5, see discussion in #119

Reply:
oh yes, of course, sorry, I overlooked the mix :)